dbt/tests/data_tests/generic_tests/expect_temporal_year_consistency.sql

```diff
@@ -0,0 +1,74 @@
+-- Generic test: expect_temporal_year_consistency
```

Comment:
```suggestion
-- Generic test: expect_consistent_years
```

Also: should this file be named `expect_consistent_years.sql` to match the name of the test macro? that would make it easier to find in the filesystem.